### PR TITLE
[Assignment2] Convert to a plugin.

### PR DESCRIPTION
**Pull request by zoe-1**
Credit to:
Myles Borins
TheAlphaNerd
https://github.com/TheAlphaNerd
This commit is modeled after his assignment 2 code below:
https://github.com/TheAlphaNerd/hueniversity/commit/c0d9d04723922632ac56be213609d9ee11e8bd6a

I learned a lot from his code.
The distributed classroom is working :-)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,30 +1,25 @@
+// ./lib/index.js 
 // Load modules
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
 var Package = require('../package.json');
```
Comment (AdriVanHoudt) [inline lib/index.js]: Not used anymore here

File: lib/index.js [hunk 2]
```diff
@@ -1,30 +1,25 @@
+// ./lib/index.js
```
Comment (AdriVanHoudt) [inline lib/index.js]: Why is this here?

Comment (zoe-1) [inline lib/index.js]: Personal habit.  When working on project in vim I think it is helpful to have this note.

File: lib/index.js (hunk 1 above)
Comment (zoe-1) [inline lib/index.js]: Thanks. I removed it.

File: lib/index.js (hunk 2 above)
Comment (AdriVanHoudt) [inline lib/index.js]: But since this is a public repo and not everyone uses vim this is unnecessary

Comment (zoe-1) [inline lib/index.js]: Good point.  I removed the comment.

Comment (AdriVanHoudt) [inline lib/index.js]: :+1:

**Comment by hueniverse** [comment]
See general comments on issue.
